install

```diff
@@ -1210,6 +1210,7 @@ else
     # not on centos
     ip_device="eth0"
     $is_ubuntu_xenial && ip_device="enp0s8"
+    $is_ubuntu_bionic && ip_device="enp0s2"
```

Comment:
I don't really Ubuntu so I can't say if this makes sense.

Reply:
It doesn't even work, TBH 😝